test/modules.js

```diff
@@ -0,0 +1,38 @@
+const test = require('tape')
+const combine = require('depject')
+const bulk = require('bulk-require')
+const path = require('path')
+
+process.env.ssb_appname = 'test'
+
+console.log(path.join( __dirname, '../modules_core'))
+
+
+console.log(`
+/////////
+// CURRENTLY TEST BROKEN
+//
+// It's bulk-require. I've tried using bulkify but it it does not seem to play well with brfs
```

Comment:
@mixmix here's a quick fix: https://github.com/ahdinosaur/patchbay/commit/e48467e995134f41edd3d38570e7d4dff82e3f54. the tests still fail, but for a different reason. :wink: